packages/backend-secret/src/secret.ts

```diff
@@ -72,7 +73,7 @@ export type SecretClientOptions = {
 };
 
 /**
- * Creates an Amplify secret client.
+ * Creates an Amplify secret client. Used in the backend lambda for fetching secrets
  */
 export const getSecretClient = (
```

Comment:
If this is being used specifically for backend lambda - should we make that more clear and rename this to getSecretClientForBackendLambda?

Reply:
We can't rename existing API without MV bump.

Reply:
we can update the implementation for this, and introduce getSecretClientForBackendLambda instead

Reply:
it does change the behavior of the existing API (i.e. from `throws SecretError` to `throws AmplifyError`) which in effect changes the API contract though not contractual in TS. I'm worried if console is already using this method and hence wanted to not touch that.